treeherder/jobs/models.py

```diff
@@ -0,0 +1,22 @@
+from django.db import models
+
+
+class JobDetail(models.Model):
+    '''
+    Represents metadata associated with a job
+    '''
+    id = models.AutoField(primary_key=True)
+    job_guid = models.CharField(max_length=50, db_index=True)
+    title = models.CharField(max_length=200, null=True)
```

Comment:
In some cases we don't have a title, for example the "purged clobber" metadata here:

https://treeherder.mozilla.org/#/jobs?repo=mozilla-inbound&selectedJob=26197592

I will update the UI to reflect this.